doc-content/enterprise-only/release-notes/rn-7.12-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,99 @@
+[id='rn-7.12-fixed-issues-ref']
+= Fixed issues in {PRODUCT} {ENTERPRISE_VERSION_LONG}
+
+{PRODUCT} {ENTERPRISE_VERSION_LONG} provides increased stability and fixed issues listed in this section.
+
+== {CENTRAL}
+
+* {CENTRAL} fails to start when a wildcard host is set to `nonProxyHosts` in `settings.xml` file [https://issues.redhat.com/browse/RHPAM-3856[RHPAM-3856]]
+* In standalone editors, the `setContent` method implementation must receive path and content [https://issues.redhat.com/browse/RHPAM-3629[RHPAM-3629]]
+* The Controller periodically pings the list of registered {KIE_SERVER}. However if this ping fails, the {KIE_SERVER} template is automatically removed [https://issues.redhat.com/browse/RHPAM-3627[RHPAM-3627]]
+* When you execute {CENTRAL} REST API through Swagger, the pop-up window for user name and password fails to open [https://issues.redhat.com/browse/RHPAM-3560[RHPAM-3560]]
+* When you create a project using the REST API, it must include project name in the response of the REST API [https://issues.redhat.com/browse/RHPAM-3519[RHPAM-3519]]
+* When you set the `org.jbpm.wb.forms.renderer.ext` system property to `true` and you try to attach a document, the *Upload* button fails to display [https://issues.redhat.com/browse/RHPAM-3515[RHPAM-3515]]
+* When you clone a Git project from outside of {CENTRAL}, you can clone the project even though it is not shown in {CENTRAL} because of lack of permissions [https://issues.redhat.com/browse/RHPAM-3375[RHPAM-3375]]
+* When you migrate a Git project from version 6.4 to version 7.8.1, you receive an error message [https://issues.redhat.com/browse/RHPAM-3232[RHPAM-3232]]
+* Editing a data object in the guided decision table fails with the null response [https://issues.redhat.com/browse/RHDM-1781[RHDM-1781]]
+* The test scenario tool is unable to execute models using imported inputs and/or decisions nodes and you receive an error message [https://issues.redhat.com/browse/RHDM-1645[RHDM-1645]]
+
+== Build and assembly
+
+* When a system logs contains an attacker controlled string value, remote code execution happens in the Java logging library of Log4j 2.x [https://issues.redhat.com/browse/RHPAM-4077[RHPAM-4077]]
```

Comment:
RHPAM-4077 not available outside of the firewall. Contact SME to make available or remove.

Reply:
Yes, I have contacted SMEs. They are going to make it public.